src/components/tag/AposTag.stories.js

```diff
@@ -0,0 +1,22 @@
+import { withKnobs, text } from '@storybook/addon-knobs';
+
+import AposTag from './AposTag.vue';
+
+export default {
+  title: 'Tags',
+  decorators: [withKnobs]
+};
+
+export const tags = () => ({
+  components: { AposTag },
+  props: {
+    label: {
+      default: text('Label', 'Neighborhood')
+    }
+  },
+  template: `
+    <AposTag
+      :label="label"
```

Comment:
Will it make sense to be passing a prop for the close icon? Thinking out loud here.

Reply:
The preemptive (before we're hitting 80 chars) line-breaks make this a bit harder to parse quickly, imo.

Reply:
make sense in that we should leave it open to receive an icon?

Reply:
Not an icon, but for whether it's closable or not. Maybe some will close on their own always?

Reply:
close on their own? these are tag UI